components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -3455,10 +3376,8 @@ public Response createNewAPIVersion(String newVersion, String apiId, Boolean def
                         apiId));
             }
             APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
-            String tenantDomain = RestApiCommonUtil.getLoggedInUserTenantDomain();
-
-            API versionedAPI = apiProvider.createNewAPIVersion(apiId, newVersion, defaultVersion);
-
+            //TODO- ORG
```

Comment:
Can we remove the Todo?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9684/commits/cc8517b20f7dbec93569cf9ab02809b6b9148cbe